make/prepare

```
@@ -3,14 +3,24 @@
 # If compling source code this dir is harbor's make dir
```

add `set -e` to fail early.

---

+1 @ninjadq is there reason for this `set +e`?